opencti-platform/opencti-front/src/private/components/settings/users/UserPopover.jsx

```diff
@@ -140,10 +140,13 @@ class UserPopover extends Component {
           TransitionComponent={Transition}
           onClose={this.handleCloseDelete.bind(this)}
         >
-          <DialogContent>
-            <DialogContentText>
-              {t('Do you want to delete this user?')}
-            </DialogContentText>
+          <DialogTitle>{t('Do you want to delete this user?')}</DialogTitle>
+          <DialogContent dividers>
+              <ul>
+                <li>{t('All notifications, triggers and digests associated with the user will be deleted.')}</li>
+                <li>{t('All investigations and dashboard where the user is the only admin, will be deleted.')}</li>
+              </ul>
+                  {t('If you want to save this information, we recommend deactivating the user instead.')}
```

Comment:
```suggestion
                  {t('If you want to keep the associated information, we recommend deactivating the user instead.')}
```

Just a detail, but it felt strange to me to use `save` as the data is already saved 😅

Reply:
Also change the Localization file 😁

Reply:
Fixed !